firestore/src/android/make_unique.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Perhaps move this file to `common`? After all, `common` also needs to compile on Android and could benefit from having `make_unique` available.